lib/middleware/preview.js

```diff
@@ -208,6 +218,12 @@ function previewMiddleware(config, req, res, next) {
  *   is used.
  * @param {Object} [config.helpers] - optional map of Handlebars helpers to
  *   register
+ * @param {Object} [config.globalModel] - optional model that will be extended
+ *   into the root object before rendering. If your ingredients contain @root
+ *   references, you'll want to specify those things here.
+ * @param {Object} [config.commonModel] - optional model that will be extended
+ *   into every ingredient's model. Useful if there are properties that all of
+ *   your ingredients assume will be in their model at all times.
```

Comment:
Ok, so what you're proposing is the ability to specify a base for the preview model as well as a base for the ingredient model. If we land this, let's call these properties `basePreviewModel` and `baseModel` or smth like that.